dev-requirements.txt

```
@@ -2,6 +2,7 @@
 black == 22.12.0
 flake8 == 5.0.4
 isort == 5.10.1
+memray == 1.6.0; platform_system != "Windows" and sys_platform != "darwin" and platform_machine != "arm64"
```

on `r6g.large` instance when I tested, `sys_platform` was `linux`, and `platform_machine` was `aarch64` (not `arm64`). I assume you tested this on AWS Glue, so you may also want to add?

```suggestion
memray == 1.6.0; platform_system != "Windows" and sys_platform != "darwin" and platform_machine != "arm64" and platform_machine != "aarch64"
```